### PR TITLE
Added additional H2/O2 reactions to the Burke kinetic libraries

### PR DESCRIPTION
**Pull request by alongd**
This PR adds six H2/O2 reactions to the Burke kinetic libraries.
Three of these reactions were so far missing from RMG (i.e., not found in libraries nor belong to any family):
- `OH + OH <=> H2 + O2`
- `H2O + O <=> H2 + O2`
- `H2O2 + O <=> H2O + O2`

Two additional reactions were added as training reactions in addition to the Burke libraries:
- `HO2 + H <=> H2O + O` (Substitution_O)
- `HO2 + H <=> H2O2` (R_Recombination)

It is noted that 'HO2 + H <=> H2O + O` is considered to have a negligible rate.

An additional reaction fits the Birad_R_Recombination family, yet since the kinetics is only available in a ThirdBody PDep form, it wasn't added as a training reaction, and only added to the libraries:
- `OH + O <=> HO2`

ref: Table III in [Burke et al.](http://dx.doi.org/10.1002/kin.20603)

RMG-Tests should be run before merging to see that these small radical training reactions make sense in their context.

These rates were purposely neglected from the H2/O2 model in the Burke et al. paper, however RMG should know what their rates are, and not estimate a much higher rate. From this point of view, only the three reactions that match a family must be added to the Burke libraries, but I guess there's no harm in being comprehensive and including all know kinetics.

Solves #237

### CODE REVIEW

**Comment by alongd** [comment]
After discussing this with @mjohnson541 , and some re-thinking, I decided to remove the two training reactions (`HO2 + H <=> H2O + O` in Substitution_O and `HO2 + H <=> H2O2` in R_Recombination) because family analogies shouldn't be drawn from these small species.

**Comment by alongd** [comment]
@mjohnson541 , this small PR is now rebased and ready for review.

File: input/kinetics/libraries/BurkeH2O2inArHe/reactions.py [hunk 1]
```diff
@@ -274,6 +274,81 @@
             Arrhenius(A=(7.59e+13, 'cm^3/(mol*s)'), n=0, Ea=(7270, 'cal/mol'), T0=(1, 'K')),
         ],
     ),
-    shortDesc = u"""Hong et al., J. Phys. Chem. A 114 (2010) 5718-5727""",
+    shortDesc = u"""Hong et al., J. Phys. Chem. A, 114:5718 (2010)""",
```
Comment (mjohnson541) [inline input/kinetics/libraries/BurkeH2O2inArHe/reactions.py]: Why was this changed?

Comment (alongd) [reply input/kinetics/libraries/BurkeH2O2inArHe/reactions.py]: Just to be consistent with the format of the other citations in this file

Comment (mjohnson541) [reply input/kinetics/libraries/BurkeH2O2inArHe/reactions.py]: Ok

**Comment by mjohnson541** [comment]
None of the kinetics changes in RMG-tests bother me too much.  Does the change in the NC model make sense to you?

**Comment by alongd** [comment]
I can't locate the RMG-tests of this branch. Could you link it here?
Since now this PR does not include any training reactions, I feel very comfortable with it, as it will only impact rates of specific reactions which are known to be relatively slow. So changes in the tests should be desired. But of course I'd like to see the tests before merging.

**Comment by mjohnson541** [comment]
RMG-tests is at:  [https://travis-ci.org/ReactionMechanismGenerator/RMG-tests/builds/334986798](url)

**Comment by alongd** [comment]
There's a rate change in two Disprop reactions in the NC test that I don't understand:
```
Non-identical kinetics!
original:
rxn: C(=N)[O](121) + O2(2) <=> HO2(9) + HNCO(64)		origin: Disproportionation
tested:
rxn: C(=N)[O](121) + O2(2) <=> HO2(9) + HNCO(64)		origin: Disproportionation
k(1bar)|300K   |400K   |500K   |600K   |800K   |1000K  |1500K  |2000K  
k(T):  |   1.48|   2.55|   3.20|   3.63|   4.17|   4.49|   4.92|   5.13
k(T):  |  -3.79|  -1.32|   0.16|   1.15|   2.38|   3.12|   4.11|   4.60
Kinetics: Arrhenius(A=(6.02974e+11,'cm^3/(mol*s)'), n=0, Ea=(5.908,'kcal/mol'), T0=(1,'K'), comment="""Estimated using template [O2b;XH_s_Rrad] for rate rule [O2b;COpri_Nrad]
Kinetics: Arrhenius(A=(1.2044e+12,'cm^3/(mol*s)'), n=0, Ea=(13.55,'kcal/mol'), T0=(1,'K'), comment="""Estimated using template [O2b;Cdpri_Rrad] for rate rule [O2b;Cdpri_Orad]
kinetics: Estimated using template [O2b;XH_s_Rrad] for rate rule [O2b;COpri_Nrad]
kinetics: Estimated using template [O2b;Cdpri_Rrad] for rate rule [O2b;Cdpri_Orad]
Non-identical kinetics!


original:
rxn: C(=N)[O](121) + O2(2) <=> HO2(9) + HNCO(64)		origin: Disproportionation
tested:
rxn: C(=N)[O](121) + O2(2) <=> HO2(9) + HNCO(64)		origin: Disproportionation
k(1bar)|300K   |400K   |500K   |600K   |800K   |1000K  |1500K  |2000K  
k(T):  |  -3.79|  -1.32|   0.16|   1.15|   2.38|   3.12|   4.11|   4.60
k(T):  |   1.48|   2.55|   3.20|   3.63|   4.17|   4.49|   4.92|   5.13
Kinetics: Arrhenius(A=(1.2044e+12,'cm^3/(mol*s)'), n=0, Ea=(13.55,'kcal/mol'), T0=(1,'K'), comment="""Estimated using template [O2b;Cdpri_Rrad] for rate rule [O2b;Cdpri_Orad]
Kinetics: Arrhenius(A=(6.02974e+11,'cm^3/(mol*s)'), n=0, Ea=(5.908,'kcal/mol'), T0=(1,'K'), comment="""Estimated using template [O2b;XH_s_Rrad] for rate rule [O2b;COpri_Nrad]
kinetics: Estimated using template [O2b;Cdpri_Rrad] for rate rule [O2b;Cdpri_Orad]
kinetics: Estimated using template [O2b;XH_s_Rrad] for rate rule [O2b;COpri_Nrad]
```

We only changed library reactions, the Disprop family shouldn't be affected.

Is the test performed vs. official/master, or an older benchmark version?

**Comment by mjohnson541** [comment]
I think it's just mixing the templates s up, if you notice they swap kinetics exactly, so the model is the same regardless.

**Comment by mjohnson541** [comment]
Is this good to merge?

**Comment by alongd** [comment]
You're right, I didn't catch this.
Yes, we're good to merge.
Thanks!
